test/unittests/libethereum/BlockChain.cpp

```diff
@@ -38,7 +38,7 @@ BOOST_FIXTURE_TEST_SUITE(BlockChainFrontierSuite, FrontierNoProofTestFixture)
 BOOST_AUTO_TEST_CASE(output)
 {
     TestBlock genesis = TestBlockChain::defaultGenesisBlock();
-    TestBlockChain bc(genesis);
```

Comment:
No changes needed in this file, all tests are already in `FrontierNoProofTestFixture` or in `MainNetworkNoProofTestFixture`